src/main/java/teammates/common/util/Templates.java

```diff
@@ -42,10 +42,8 @@ public static String populateTemplate(String template, String... keyValuePairs)
                 FileHelper.readResourceFile("instructorEmailFragment-courseJoin.html");
         public static final String USER_FEEDBACK_SESSION =
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSession.html");
-        public static final String FRAGMENT_CLOSED_SESSION_ADDITIONAL_INSTRUCTIONS =
-                FileHelper.readResourceFile("userEmailFragment-sessionClosedAdditionalInstructionsFragment.html");
-        public static final String USER_FEEDBACK_SESSION_CLOSING =
-                FileHelper.readResourceFile("userEmailTemplate-feedbackSessionClosing.html");
+        public static final String USER_FEEDBACK_SESSION_CLOSING_AND_REMINDER =
+                FileHelper.readResourceFile("userEmailTemplate-feedbackSessionClosingAndReminder.html");
```

Comment:
`ClosingAndReminder` is too verbose. Just `Reminder` will do since the purpose of the email is to remind the students anyway.

Actually, since this template and the `USER_FEEDBACK_SESSION` template differs only in the "No action required" portion, let's see if you can push this even further: try to make everything come from only one template file and let the "No action required" portion be generated on demand.

Reply:
Okay, I will do that. That would be great, as I was also thinking that there should be a more elegent solution to generate these emails than to keep two templates that differs only in one line.